<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightWindow</class>
 <widget class="QMainWindow" name="FlightWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>834</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Авиарейсы</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1273</width>
      <height>132</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="5">
      <widget class="QLabel" name="time_label">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string>Время
отправления</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="state_label">
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>151</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string>Статус</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="num_flight_label">
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string>Номер
авиарейса</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="all_place_label">
       <property name="minimumSize">
        <size>
         <width>132</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>132</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string>Количество
мест
всего</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="company_label">
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>90</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Название
авиакомпании</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="to_label">
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string>Аэропорт
прибытия</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="date_label">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string>Дата
отправления</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="from_label">
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string>Аэропорт
отправления</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="num_flight_edit">
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="free_place_label">
       <property name="minimumSize">
        <size>
         <width>132</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>132</width>
         <height>90</height>
        </size>
       </property>
       <property name="text">
        <string>Количество
свободных
мест</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="company_edit">
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="from_edit">
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="to_edit">
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QComboBox" name="state_box">
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>151</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>По расписанию</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Регистрация</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Посадка</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Вылетел</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Задерживается</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Отменён</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="date_edit">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="time_edit">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLineEdit" name="all_place_edit">
       <property name="minimumSize">
        <size>
         <width>132</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>132</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QLineEdit" name="free_place_edit">
       <property name="minimumSize">
        <size>
         <width>132</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>132</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>730</y>
      <width>1271</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="add_button">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_button">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="change_button">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Авиарейсы</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableView" name="flight_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>1271</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>139</number>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1290</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
